tests/Microsoft.Bot.Builder.ApplicationInsights.Tests/TelemetryWaterfallTests.cs

```diff
@@ -66,14 +61,26 @@ public async Task WaterfallWithCallback()
 
             var dialogState = convoState.CreateProperty<DialogState>("dialogState");
             var dialogs = new DialogSet(dialogState);
-            var telemetryClient = new Mock<IBotTelemetryClient>(); ;
-            var waterfallDialog = new WaterfallDialog("test", new WaterfallStep[]
+            var telemetryClient = new Mock<IBotTelemetryClient>();
+            var steps = new WaterfallStep[]
```

Comment:
Can this be replaced by the method NewWaterfall()? I think this will reduce duplicated code